src/index.html

```diff
@@ -1,52 +1,35 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="fr">
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Exercice 1</title>
     <link rel="stylesheet" href="style.css">
+    <title>Excercice</title>
 </head>
 <body>
-    <section id="header">
-        <div id="titre">
-            <h1>Conception Web en 4 minutes</h1>
-            <div class="text">
-                <p>
-                    <a href="https://github.com/LaurieBdt">par Laurie Bodot</a>
-                </p>
-            </div>
-        </div>
-    </section>
+    <form id="home">
+        <section id="titre">
+                <h1>Login</h1>
```

Comment:
```suggestion
                <h1 class="card-title">Login</h1>
```

Traduit en français stp